frontend/views/containers/chatroom/ChatMembersAllModal.vue

```diff
@@ -0,0 +1,391 @@
+<template lang='pug'>
+modal-base-template.has-background(ref='modal' :fullscreen='true' :a11yTitle='L("Channel members")')
+  .c-container
+    .c-header
+      div
+        i18n.is-title-2.c-title(
+          tag='h2'
+        ) Members
+        .c-description {{ attributes.name }} ∙ {{attributes.private ? L("Private channel") : L("Public channel")}}
+
+    .card.c-card
+      search(
+        :placeholder='L("Search...")'
+        :label='L("Search")'
+        v-model='searchText'
+      )
+
+      .c-member-count.has-text-1(
+        v-if='searchText && searchCount > 0'
+        data-test='memberSearchCount'
+        v-safe-html='resultsCopy'
+      )
+
+      i18n.c-member-count.has-text-1(
+        v-if='searchText && searchCount === 0'
+        tag='div'
+        :args='{searchTerm: `<strong>${searchText}</strong>`}'
+      ) Sorry, we couldn't find anyone called "{searchTerm}"
+
+      .c-list-to-add
+        .is-subtitle
+          i18n(
+            tag='h3'
+            :args='{  nbMembers: chatRoomUsersInSort.length }'
+          ) Channel members ({nbMembers})
+
+        transition-group(
+          v-if='addedMembers'
+          name='slide-list'
+          tag='ul'
+          data-test='joinedChannelMembersList'
+        )
+          li.c-search-member(
+            v-for='{username, displayName, departedDate} in addedMembers'
+            :key='username'
```

Comment:
Yes. We could use `index` as a key. But `username` is also fine as it can not be duplicated.
@pieer, do you think we should use `index` as a key for all the loops in order to keep consistency?

Reply:
Don't change everything everywhere, but just keep good practice from now. on :)

Reply:
👍